helper-core/src/main/java/org/example/domain/auth/spi/JwtPort.java

```diff
@@ -0,0 +1,8 @@
+package org.example.domain.auth.spi;
+
+import org.example.domain.auth.model.Role;
+
+public interface JwtPort {
+    String GenerateAccess(String userId, Role role);
+    String GenerateRefresh(String userId);
+}
```

Comment:
인터페이스 에서 메서드를 이렇게 붙여놓으면 IDE는 사람마다 다르긴해도 레포지토리에서 볼때에는 굉장히 더러워 보이는 코드 스타일이라 아래 한줄은 붙이되 다른 모든 줄은 행을 띄우는건 어떨까요

```suggestion
public interface JwtPort {

    String GenerateAccess(String userId, Role role);
    
    String GenerateRefresh(String userId);
}
```

Reply:
음 좋은것 같네요! 반영하겠습니다! 앞으로 그렇게 짜도록 노력하겠습니다.

Reply:
전체적인 코드스타일을 신경써달라는 말이었는데 이것만 바꾸시면 어캄;

Reply:
@songju7920 @songju7920 @songju7920 @songju7920 @songju7920 @songju7920 @songju7920 @songju7920

Reply:
아니 알겠습니다.. 진정하세요

Reply:
interface 형식 수정완료!